.github/actions/setup-oracle/action.yml

```diff
@@ -0,0 +1,13 @@
+name: Create Oracle container
+runs:
+  using: composite
+  steps:
+    - shell: bash
+      run: |
+        set -x
+        docker run \
+          -e "ORACLE_PASSWORD=pass" \
+          -p 1521:1521 \
+          --name oracle \
+          -h oracle \
+          -d gvenzl/oracle-xe:18
```

Comment:
This should use `slim` aka 21.3, not the obsolete 18c version.

Reply:
I just saw your https://github.com/php/php-src/issues/8346#issuecomment-1096600649 comment, will update shortly.